packages/fxa-settings/src/components/RecoveryKeySetupDownload/index.tsx

```diff
@@ -22,9 +21,8 @@ interface RecoveryKeySetupDownloadProps {
   email: string;
 }
 
-// TODO FXA-8305
 const spinner = (
-  <LoadingSpinner className="bg-grey-20 flex items-center flex-col justify-center h-screen select-none" />
+  <LoadingSpinner className="bg-grey-20 flex items-center flex-col justify-center select-none" />
```

Comment:
This fixes:
![image](https://github.com/user-attachments/assets/a5812fe4-161b-4a7b-b1f3-0fae200e0556)